planq/assetlist.json

```diff
@@ -139,6 +139,175 @@
           }
         }
       ]
+    },
+    {
+      "description": "The token of Anubis.",
+      "extended_description": "Anubis is a Meme coin created via the Physica launchpad.",
+      "denom_units": [
+        {
+          "denom": "erc20/0xF630B96760B91f6FCbE8b3C24f2786AcA3D03648",
+          "exponent": 0
+        },
+        {
+          "denom": "anubi",
+          "exponent": 18
+        }
+      ],
+      "base": "erc20/0xF630B96760B91f6FCbE8b3C24f2786AcA3D03648",
+      "name": "Anubis",
+      "display": "anubi",
+      "symbol": "ANUBI",
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/planq/images/anubis.png"
+      },
+      "images": [
+        {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/planq/images/anubis.png",
+          "theme": {
+            "primary_color_hex": "#7f0f70"
```

Comment:
I don't see this color at all in any of the images. If not going to be used properly, please remove